test/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.join(
+    os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+    'src'))
```

Comment:
Nose seems to be modifying the path so that the module in the repository directory is prefixed on the path, and pytest doesn't seem to do that (at least when a `src` subdirectory is used).

This is the approach that `catkin_pkg` uses, and it seems to work fine here as well.